src/scripts/main.js

```diff
@@ -1,3 +1,47 @@
 'use strict';
 
-// write code here
+const title = document.querySelector('thead tr');
+const tbody = document.querySelector('tbody');
+const list = document.querySelectorAll('tbody tr');
+
+title.addEventListener('click', (element) => {
+  if (element.target.tagName === 'TH') {
+    let columnIndex;
+
+    if (element.target.textContent === 'Name') {
+      columnIndex = 1;
+    } else if (element.target.textContent === 'Position') {
+      columnIndex = 2;
+    } else if (element.target.textContent === 'Age') {
+      columnIndex = 3;
+    } else if (element.target.textContent === 'Salary') {
+      columnIndex = 4;
```

Comment:
The logic for determining `columnIndex` is hardcoded based on the text content of the headers. If the table structure changes or if there are additional columns, this logic will need to be updated. Consider using a more dynamic approach, such as mapping headers to indices based on their position in the DOM.